### DISCUSSION TITLE
Help With Animating Objects

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#105] [Help / Questions] [answered]
I'm following the animation example to change the animation one of my meshes from "Idle" to "Running". However, when I do so, the resulting animation is an equal blend of the idle animation and of the run animation. Is there any way I can completely turn the idle animation off and solely switch to the new animation?

**Comment by N8python** [comment]
After some trial and error, I found that the following code worked to disable an animation:
```
 this.enemy.animation.mixer._actions[0].setEffectiveWeight(0);
```
Is there a better alternative?

**Comment by yandeu** [comment, marked as answer]
I don't know much about animations. 
Hope these links help:
- [AnimationMixer](https://threejs.org/docs/#api/en/animation/AnimationMixer)
- [AnimationAction](https://threejs.org/docs/#api/en/animation/AnimationAction)

**Reply by N8python** [reply]
Awesome, thanks!
Sorry for all the questions - just getting started with the library. Thanks for you help and availability.

**Reply by yandeu** [reply]
You're welcome 😁

Glad someone uses it!

**Reply by N8python** [reply]
More people should use it. It's the best JavaScript Physics library I've come across so far.
